### ISSUE TITLE
Uploaded tracks not showing up anywhere

### ISSUE THREAD

**Issue by bergziege** [Flomp/wanderer#3]
Hi, just found this very nice app via reddit and had to try it out.
Installed it via the provided docker-compose and averything seems to run fine after registering a new user ... but:

After saving a new trail (which I get a success message for) its not showig up anywhere. Not in any search and not on the map sor anywhere else .... but

If I use Firefox and navigate from the Trails page to the Home page all my tracks show up for a fraction of a second before the new page is being shown.
Edge not showing anything at all (even when switching pages).

I am not using a reverse proxy yet and not getting any error messages in the logs (at least the ones I can see via portainer).

Another question: Where are my uploaded gpx files saved? I was under the impression that they are saved as plain files but can not find them. Would like to have the uploaded files as plain files somewhere for easier backup.

**Comment by Flomp** [comment]
Hi, thanks for checking it out.
That sounds a bit like transferring the trail to the search index did not work properly. Is the trail visible on front page when you scroll down a little?
Is the db container able to communicate with the search container?

Could you please try to exec into the db container and run:
`apk add curl && curl -H "Authorization: Bearer $MEILI_MASTER_KEY" $MEILI_URL/indexes/trails/search`

Regarding your other question: the gpx file is stored along with all the other data in the db container. You can download them at any point from wanderer (once your trails start showing up).

**Comment by bergziege** [comment]
Hi, thanks for the quick response.

Yes, I can see my trails on the front page. I also see them when searching for them on the front page.

Executing the given command seems to not find `$MEILI_URL` as its outputting `curl: (2) no URL specified` after installing curl succesfully.

My docker compose is as follows. "Just" changed the external facing ports but left the internal urls untouched.

```
version: '3'

x-common-env: &cenv
  MEILI_URL: http://search:7700
  MEILI_MASTER_KEY: dontknow

services:
  search:
    container_name: wanderer-search
    image: flomp/wanderer-search
    environment:
      <<: *cenv
      MEILI_NO_ANALYTICS: true
    ports:
      - 7200:7700
    networks:
      - wanderer
    volumes:
      - ./data/data.ms:/meili_data/data.ms
    restart: unless-stopped
    healthcheck:
      test: curl --fail http://localhost:7700/health || exit 1
      interval: 15s
      retries: 10
      start_period: 20s
      timeout: 10s
  db:
    container_name: wanderer-db
    image: flomp/wanderer-db
    depends_on:
      search:
        condition: service_healthy
    environment:
      <<: *cenv
    ports:
      - "7201:8090"
    networks:
      - wanderer
    restart: unless-stopped
    volumes:
      - ./data/pb_data:/pb_data
  web:
    container_name: wanderer-web
    image: flomp/wanderer-web
    depends_on:
      search:
        condition: service_healthy
      db:
        condition: service_started
    environment:
      <<: *cenv
      ORIGIN: http://192.168.2.105:7202
      PUBLIC_POCKETBASE_URL: http://db:8090
    ports:
      - "7202:3000"
    networks:
      - wanderer
    restart: unless-stopped

networks:
  wanderer:
    driver: bridge
```

**Comment by Flomp** [comment]
I think I found the problem. I suspect this is the same problem as [#10](https://github.com/Flomp/wanderer/issues/10). Could you please quickly confirm that your trail is either longer than 20km or has an elevation gain of more than 4000m? In that case I will publish the fix shortly.

**Comment by Flomp** [comment]
I just released v0.1.1. The issue should be resolved now.

**Comment by bergziege** [comment]
Sry for the late response. 0.1.1 fixed the issue as my "Trail" was definitely longer than 20km.
All my trails now show up on the search as well as on the map. Thank you!

My use cases for the app: 
- Having a catalogue of cycling routes with lenghts of about 50-200km to quickly pick training rides based on distance and climbing if I am too lazy to create new ones. <- thats the one I nearly started creating an app like wanderer by myself.
- MAYBE as a catalog for multi day trips (but those would be splitted to <200km segments anyway)
- NOT as a ride history as I have those on the respective tracking providers page like garmin-connect or strava

My most loved features based on my use cases would be:
- Not limit the distance filter to 20+ but to the max value in the db
- Clicking a route on the map should select the route / Hover should highlight
